source/articles/releasenotes/Unreleased-Changes.md

```diff
@@ -6,3 +6,60 @@ This page contains changes and fixes that were not inluded in any release yet an
 
 none yet
 
+## ASP.NET Core support
+LINQ To DB now has support for ASP.NET Dependancy injection. Here's a simple example of how to add it to dependancy injection
```

Comment:
Let's just remove this file and move content to main article and add something like `Available since: 3.0.0-rc.0` on top. Now I don't think this was a good idea to add this